specifyweb/frontend/js_src/lib/components/PickLists/fetch.ts

```diff
@@ -53,7 +55,10 @@ async function unsafeFetchPickList(
       // Pick list does not exist
       return undefined;
     if (!hasToolPermission('pickLists', 'read')) return undefined;
-    pickList = await rawFetchPickList(pickListName, true);
+    pickList =
+      pickListName === 'systemGeologicPicklist'
+        ? geologicPickList
+        : await rawFetchPickList(pickListName, true);
```

Comment:
Name check not needed after defining a CO -> Age picklist

```suggestion
    pickList = await rawFetchPickList(pickListName, true);
```